src/components/TabFilter.tsx

```diff
@@ -37,7 +37,7 @@ const ButtonContainer = styled.ul`
   display: flex;
   gap: ${spacing.xsmall};
   padding: ${spacing.xsmall};
-  border-radius: 12px;
+  border-radius: ${spacing.small};
```

Comment:
Jeg syntes egentlig ikke spacing skal brukes for border-radius, heller bare beholde det som var hvis vi ikke kan bruke predefinert border-radius 🤔 men hvis andre er uenige så kan det sikkert beholdes

Reply:
Dette er en feil, det gikk litt kjapt i svingene